_assets/scripts/output/script1.txt

```diff
@@ -0,0 +1,5 @@
+*----------1.2
+**---------1.2
+***--------1.2
+****-------1.2
+*****------1.2
```

Comment:
What are these changes?
(I don't think these are used at all. Could you try to delete this file (and `script2.svt`) and see that it still works locally?)

Reply:
I didn't change something in this file, which is why I was also wondering why changes were displayed. I deleted those files now and the website works well locally. Do I need to open up another pull request so that the new changes are included?

Reply:
Great! 
> I see that this pull request got updated by the new changes so I think I don't need to open a new pull request.

Yes, all changes to the branch in your repo (in this case `saidharb#main`) will update this PR.
(The standard/typical workflow is that you create a new branch on your fork (`git checkout -b fix_collaps`) and push your changes to that (`git push fork fix_collaps`) (if your git remote to your fork is called "fork"), and make a pull request to this repo from that branch. That way, you can have multiple open PRs at the same time)